langchain_dappier/retrievers.py

```
@@ -0,0 +1,212 @@
+            query: String to find relevant documents for
+            run_manager: The callabacks handler to use
+        Returns:
+            List of relevant documents
+        """
+        try:
+            from dappier import Dappier
+        except ImportError:
+            raise ImportError(
+                "Dappier python package not found."
+                "Please install it with `pip install dappier`"
+            )
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)